src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/MethodKindEx.cs

```diff
@@ -14,11 +14,11 @@ namespace Microsoft.CodeAnalysis.Shared.Extensions
     [SuppressMessage("CodeQuality", "IDE0051:Remove unused private members", Justification = "Private constants are used for static assertions.")]
     internal static class MethodKindEx
     {
-        public const MethodKind FunctionPointerSignature = (MethodKind)18;
+        //public const MethodKind FunctionPointerSignature = (MethodKind)18;
```

Comment:
Did you mean to leave this commented out?

Reply:
Yes, each of these lightup types preserves one commented entry as a template.